osbs/utils.py

```diff
@@ -188,29 +191,36 @@ def buildconfig_update(orig, new, remove_nonexistent_keys=False):
 
 
 @contextlib.contextmanager
-def checkout_git_repo(git_uri, git_ref, git_branch=None):
+def checkout_git_repo(git_uri, git_ref, git_branch=None, max_retries=GIT_MAX_RETRIES,
+                      cooldown=GIT_BACKOFF_FACTOR):
```

Comment:
can we just name this kwarg `backoff_factor`? :)

Reply:
I believe the name here is good to show the variable is related to git. We also have similar variables for HTTP

Reply:
Oh, I see what you mean. I will change it :)